viscy/representation/lca.py

```diff
@@ -188,6 +221,13 @@ def train_and_test_linear_classifier(
         Batch size, by default 1024
     lr : float, optional
         Learning rate, by default 1e-3
+    use_smote : bool, optional
+        Whether to apply SMOTE to the training data, by default False
+    save_predictions : bool, optional
+        Whether to save predictions to CSV, by default False
+    csv_path : str, optional
+        Path to save predictions CSV, by default None
+    merged_df : pd.DataFrame, optional
```

Comment:
What should this be?

Reply:
csv_path is where to same predictions and merged_df is the dataframe of initial input data. You can see the example script to see functionality of this.

Reply:
Can you include the above information in the docstring?